config/production.json

```diff
@@ -2,4 +2,13 @@
   "server": {
     "port": 9000
   }
+  "db" : {
```

Comment:
None of this is necessary in `production.json` as it doesn't differ from the defaults.

Reply:
True, but why not leave the values in for documentation purposes? They'll have to be filled in for production someday.

Reply:
Because they're still in `default.json` for reference.